proprietary/tokens/src/components/amsterdam/alert.tokens.json

```diff
@@ -1,36 +1,42 @@
 {
   "amsterdam": {
     "alert": {
-      "background-color": { "value": "{amsterdam.color.yellow}" },
-      "border": { "value": "4px solid {amsterdam.color.yellow}" },
+      "border-width": { "value": "{amsterdam.border-width.xl}" },
+      "border-style": { "value": "solid" },
```

Comment:
Not added to common border tokens?

Reply:
Yes it was, in a69bb9acd95fce6ef0a786d614b8f319a34ba34c.

Reply:
And in the meantime we’re no longer using this token.